osc-domain/src/main/java/org/osc/core/broker/model/entities/virtualization/k8s/Label.java

```diff
@@ -29,15 +29,23 @@
 import javax.persistence.Table;
 
 import org.osc.core.broker.model.entities.BaseEntity;
+import org.osc.core.broker.model.entities.virtualization.ProtectionEntity;
 import org.osc.core.broker.model.entities.virtualization.SecurityGroupMember;
+import org.osc.core.broker.model.entities.virtualization.SecurityGroupMemberType;
 
 @SuppressWarnings("serial")
 @Entity
 @Table(name = "LABEL")
-public class Label extends BaseEntity {
-    @Column(name = "value", nullable = false)
+public class Label extends BaseEntity implements ProtectionEntity {
+    public Label() {
```

Comment:
move constructor after variables and make it package private or private

Reply:
the persistence frameworks requires it to be public.

Reply:
package private should work. it just needs a default constuctor...